File: benchmarks/cbrt-100K/Main.java
public class Main {

    public static void main(String[] args) {
        long before = System.nanoTime();

        for (int i = 0; i < 100_000; i++) {
            Math.cbrt(i);
        }

        long after = System.nanoTime();

        System.out.println(after - before);
    }

}

File: benchmarks/cbrt-100K/main.zig
const std = @import("std");

pub fn main() !void {
    const before: i128 = std.time.nanoTimestamp();

    for (0..99999) |i| {
        const j = @as(f32, @floatFromInt(i));
        _ = std.math.cbrt(j);
    }

    const after: i128 = std.time.nanoTimestamp();

    std.debug.print("{}", .{after - before});
}